.github/workflows/arm64.yml

```diff
@@ -38,9 +38,8 @@ jobs:
         go-version: '1.21'
         check-latest: true
     - name: Install buf cli
-      uses: bufbuild/buf-setup-action@v1
-      with:
-        github_token: ${{ github.token }}
+      run: |
+        go install github.com/bufbuild/buf/cmd/buf
```

Comment:
Don't you need the `@version` ? e.g., `@latest`

Reply:
I think it should use the version from the go.mod file? 🤞

Reply:
Ye, momentary lapse of judgment.

Reply:
Didn't work - added version.